src/qt/pivx/settings/settingsnetworkwidget.cpp

```diff
@@ -25,7 +25,7 @@ SettingsNetworkWidget::SettingsNetworkWidget(PIVXGUI* _window, QWidget *parent)
     setCssBtnPrimary(ui->pushButtonSave);
     setCssBtnSecondary(ui->pushButtonReset);
 
-    connect(ui->pushButtonSave, SIGNAL(clicked()), parent, SLOT(onSaveOptionsClicked()));
+    connect(ui->pushButtonSave, &QPushButton::clicked, [=] { Q_EMIT saveSettings(); });
```

Comment:
Even if this seems to be disconnected now, we should grab the object by reference here too
```cpp
connect(ui->pushButtonSave, &QPushButton::clicked, [this] { Q_EMIT saveSettings(); });
```

Reply:
done